hab_gui/widgets/alias_button.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from Qt import QtCore, QtGui, QtWidgets
+
+
+class AliasButton(QtWidgets.QToolButton):
+    """Create a QToolButton which will launch a specified alias via a subprocess.
+
+    Args:
+        cfg (hab.parsers.flat_config.FlatConfig): The config object which contains URI related data.
+        alias_name (str): The alias name passed from AliasButtonGrid.
+        parent Define a parent for this widget.
```

Comment:
This is not a valid docstring for parent. It should be something like:
`parent (Qt.QtWidgets.QWidget, optional): Define a parent for this widget.`

This note applies to all of the other places this was used.